script/Common/00_00.txt

```diff
@@ -1469,7 +1534,9 @@ With your eyes, it becomes dangerously easy to take a life."
 [sha:c789c414b14caf6bcc14b69637ae35b52e5fb62a]{
 -- Page 110, Offset 400.
 -- それに、なんだか胸がいたいんだ。……ごめんね先生。もう、二度とあんなことはしないから」
-Besides, my chest is starting to hurt... I'm sorry, Sensei. I won't do it again."
+// Assuming 胸がいたむ is figurative
+I feel terrible about it, too...
```

Comment:
maybe. However, it _is_ established later that stuff related to the lines of death causes his scar to hurt.